ejb3/src/main/java/org/jboss/as/ejb3/timerservice/TimerImpl.java

```diff
@@ -605,6 +632,7 @@ public static Builder builder() {
         protected TimerState timerState;
         protected boolean persistent;
         protected boolean newTimer;
+        protected Thread executingThread;
```

Comment:
AFAICT this field is never read. The TimerImpl constructor does not use it.

Reply:
Fixed. I missed it in the TimerImpl constructor.

Reply:
Why would there be an executing thread when the timer is being constructed? Won't this always be null?